Assets/Scripts/EOSAchievementManager.cs

```diff
@@ -301,34 +249,35 @@ private static void QueryAchievements(ProductUserId productUserId, OnQueryDefini
         /// The ProductUserId that corresponds to the player for whom the
         /// achievements are being queried.
         /// </param>
-        /// <param name="callback">
-        /// Invoked when the request from the server has completed - whether
-        /// successful or not.
-        /// </param>
-        private static void QueryPlayerAchievements(ProductUserId productUserId, OnQueryPlayerAchievementsCompleteCallback callback)
+        /// <returns>
+        /// List of PlayerAchievement objects.
+        /// </returns>
+        private Task<List<PlayerAchievement>> QueryPlayerAchievementsAsync(ProductUserId productUserId)
```

Comment:
If the above Task function should be async, this should also be async. Otherwise, ignore this comment.

Reply:
As before - does not need to be async, as it makes no await calls within it. This does not prevent it from being `await` called.

Reply:
Copy that, you've got it right here.